### ISSUE TITLE
CLI Upgrade: stat: illegal option -- c

### ISSUE THREAD

**Issue by tuaris** [owncloud/core#28758]
### Steps to reproduce
1. Upgrade to 10.0.2
2. Run upgrade in browser
3. Run upgrade in console (php occ upgrade)

### Expected behavior
Upgrade should complete

### Actual behavior
Browser based upgrade fails.  Console based upgrade does not complete.  Error messages shown are

```
stat: illegal option -- c
usage: stat [-FLnq] [-f format | -l | -r | -s | -x] [-t timefmt] [file ...]
```

### Server configuration
**Operating system**: FreeBSD 10.3

**Web server:** Apache 2.4

**Database:** MySQL 5.6

**PHP version:** 5.6

**ownCloud version:** 10.0.2

**Updated from an older ownCloud or fresh install:** Update from 9.x

**Where did you install ownCloud from:** FreeBSD Ports

**Signing status (ownCloud 9.0 and above):**

Unable to sign in.

**Comment by DeepDiver1975** [comment]
The code in question is here

https://github.com/owncloud/core/blob/bafdb425f350b19de79f5c522388df5ef878b800/lib/private/LargeFileHelper.php#L148-L159

**Comment by DeepDiver1975** [comment]
Looks like your OS is detected as linux and not as bsd ...

Please run this in your terminal

```sh

php -r "echo php_uname('s') . PHP_EOL;"

**Comment by DeepDiver1975** [comment]
FYI: This code is triggered mainly because you are running a 32 bit and not 64 bit architecture

**Comment by tuaris** [comment]
```
 php -r "echo php_uname('s') . PHP_EOL;"
FreeBSD
```

This is 32-bit (yes I know I should upgrade to 64-bit ).

**Comment by phil-davis** [comment]
filemtime checks have a ``stat -c`` command that only has the case for Linux. Maybe #28759 will help?

**Comment by kdslkdsaldsal** [comment]
Ref: https://github.com/owncloud/core/pull/28761

**Comment by phil-davis** [comment]
Should be fixed with #28759 and backport to stable10 #28790
